Upgrade client to 0.18.2

JonoPrest opened the pull request:
Will only work once this is merged and released:

https://github.com/enviodev/hypersync-client-rust/pull/66

JasoonS commented on cargotree:
I'm not familiar with this type of file - do we want this committed?

JonoPrest replied:
Whoops, didn't mean to commit. Was just looking at the cargo tree to figure out which packages were causing dependency clashes.  Will remove

JasoonS commented:
I believe merging this will break these docs: https://enviodev.github.io/hypersync-client-node/

JasoonS commented:
Will revert that. Otherwise I think this looks fine as a start.

JonoPrest commented:
@JasoonS I've fixed it, I simply ran the `yarn docs` script but it was configured to use markdown plugin. I changed it to just use the basic one and now it generated html
